templates/web/base/admin/reports/edit.html

```diff
@@ -181,21 +167,49 @@
                 <span>zoom</span>
             </a>
         </div>
-        <input type="submit" name="rotate_photo_[% loop.index %]" value="[% loc('Rotate Left') %]">
-        <input type="submit" name="rotate_photo_[% loop.index %]" value="[% loc('Rotate Right') %]">
+        <input class="btn" type="submit" name="rotate_photo_[% loop.index %]" value="[% loc('Rotate Left') %]">
+        <input class="btn" type="submit" name="rotate_photo_[% loop.index %]" value="[% loc('Rotate Right') %]">
         <br><input type="checkbox" id="remove_photo_[% loop.index %]" name="remove_photo_[% loop.index %]" value="1">
         <label class="inline" for="remove_photo_[% loop.index %]">[% loc("Remove photo (can't be undone!)") %]</label></li>
     </li>
   [% END %]
 </ul>
 </li>
 [% END %]
+</ul>
 
-<li><label class="inline-text" for="closed_updates">[% loc('Closed to updates') %]:</label>
-    <input type="checkbox" id="closed_updates" name="closed_updates"[% ' checked' IF problem.extra.closed_updates %]></li>
+<div class="clearfix full-width" style="background-color: #eee; padding: 1em 1em 0;">
+  [% IF allowed_pages.user_edit %]
+    <p style="float:right"><a href="[% c.uri_for_action('/admin/users/edit', [ problem.user.id ]) %]">[% loc('Edit user') %]</a>
+  [% END %]
+
+<ul class="plain-list">
+<li><label class="inline-text" for="anonymous">[% loc('Anonymous:') %]</label>
+<select class="form-control" name="anonymous"  id="anonymous">
+<option [% 'selected ' IF problem.anonymous %]value="1">[% loc('Yes') %]</option>
+<option [% 'selected ' IF !problem.anonymous %]value="0">[% loc('No') %]</option>
+</select></li>
+<li><label class="inline-text" for="name">[% loc('Name:') %]</label>
+    [% loc('(for this report)') %]
+    <input type='text' class="form-control" name='name' id='name' value='[% problem.name | html %]'></li>
 
+<li><label class="inline-text" for="username">[% loc('User:') %]</label>
```

Comment:
Should this be `[% loc('Username:') %]`?

Reply:
We don't use the word username anywhere, so probably shouldn't start now.